### PR TITLE
Initial 0.0.1 version of plugin

### PR DESCRIPTION
**Pull request by dbaggott**
Complete working plugin

### CODE REVIEW

File: src/main/java/com/cafepress/newrelic/plugin/dropwizard/health/HttpHealthCheckProbe.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    private final ObjectMapper objectMapper;
+
+    public HttpHealthCheckProbe(WebTarget adminWebTarget, ObjectMapper objectMapper) {
+        checkNotNull(adminWebTarget);
+        checkNotNull(objectMapper);
+        this.webTarget = adminWebTarget.path("/healthcheck");
+        this.objectMapper = objectMapper;
+    }
+
+    public HealthCheckResponse probe() {
```
Comment (szou-cp) [inline src/main/java/com/cafepress/newrelic/plugin/dropwizard/health/HttpHealthCheckProbe.java]: `@override`

File: src/main/java/com/cafepress/newrelic/plugin/dropwizard/health/HttpHealthCheckProbe.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        try {
+            Response webResponse = webTarget.request(MediaType.APPLICATION_JSON_TYPE).get();
+            HealthCheckResponse healthCheckResponse = parseHealthCheck(webResponse);
+            webResponse.close();
```
Comment (szou-cp) [inline src/main/java/com/cafepress/newrelic/plugin/dropwizard/health/HttpHealthCheckProbe.java]: if we have to close it, better wrapped it in `try ... finally ....`
arguably you are already doing ` catch (Exception ex)` in  `parseHealthCheck` and it is not necessarily here.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,69 @@
...
+messages within NR.
+
+Multiple Dropwizard applications can be monitored from the same plugin agent.  The plugin polls all configured Dropwizard 
+applications every minute and publishes the appropriate data into NR.  The plugin handles failures to connect to or 
+receive a response from a Dropwizard application; only applications that provide a valid 200 status code response are
+considered healthy.
+
+## New Relic Dashboard UI
+
+![dashboard ui](http://cafepressdev.github.io/newrelic-dropwizard-plugin/images/nr-dashboard-ui.png)
```
Comment (szou-cp) [inline README.md]: not critical but better to hide those labels like "Elasticsearch", "Processor" ...

Comment (dbaggott) [inline README.md]: Yeah, I considered that and decided not to worry about it -- there's nothing important there.

**Comment by dbaggott** [comment]
@szou-cp added the close() on finally (and a few other minor things) -- I think you're right that it was fine as-is but this is safer.

**Pull request merged**